test/Dialect/Torch/refine-public-return.mlir

```diff
@@ -3,10 +3,11 @@
 // CHECK-LABEL:   func.func @basic(
 // CHECK-SAME:                %[[ARG:.*]]: !torch.vtensor<[2,3,?],f32>) -> !torch.vtensor<[2,3,?],f32> {
 // CHECK:           return %[[ARG]] : !torch.vtensor<[2,3,?],f32>
-func.func @basic(%arg0: !torch.vtensor<[2,3,?],f32>) -> !torch.tensor {
+func.func @basic(%arg0: !torch.vtensor<[2,3,?],f32>) -> !torch.optional<tensor> {
```

Comment:
Can you add new tests instead of changing these? Lit tests should ideally have a single responsibility regarding what they are testing. This makes debugging test failures easier and also makes it clear from looking at the test what is being checked.